src/lib.rs

```diff
@@ -143,3 +195,12 @@ enum Reactor {
     Handle(reactor::Handle),
     Remote(reactor::Remote),
 }
+
+cfg_if! {
+    if #[cfg(feature = "tls")] {
+        extern crate tokio_tls;
+        pub extern crate native_tls;
+
+        pub use client::tls::TlsClientContext;
```

Comment:
Related to my other comment, I'm starting to feel more strongly that there should be a `mod tls` at the crate root, within which is a client module.